### ISSUE TITLE
Are there circumstances in which an overlay tool could be seen as providing a conforming alternate version for a non-conforming website?

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/wcag#2748]
Public bodies increasingly fall for promises of accessibility overlay providers that putting an overlay on their sites will make them WCAG 2.1 conformant in a matter of hours. In all cases we know of this is not true. Moreover, most overlay tools we have investigated have themselves some or many accessibility barriers in the settings dialog offered to users for customisation of the site. But this is not the topic I would like the WG to clarify here.

This question is this: 
**In the (unlikely) case that an overlay tool is actually able to successfully remedy all WCAG fails of a site via a specific set of user settings, can this state be considered a conforming alternate version (CAV)?**

Looking at the Conformance requirements and the definition of CAV, the situation seems a bit fuzzy.

Section [5.2. Conformance Requirements](https://www.w3.org/TR/WCAG21/#conformance-reqs) talks about one specific version ("…or a conforming alternate version is provided")  and [the definition of conforming alternate version](https://www.w3.org/TR/WCAG21/#dfn-conforming-alternate-version) also talks about "the conformaing alternative version" (singular) . One note explicitly qualifiies "One version would need to be fully conformant in order to meet conformance requirement 1."

Where it gets fuzzy is with [the last note of the CAV definition](https://www.w3.org/TR/WCAG21/#h-note-72), which says:
> "Setting user preferences within the content to produce a conforming version is an acceptable mechanism for reaching another version as long as the method used to set the preferences is accessibility supported."

There is no stipulation that just **one** switch (say a style switcher) or **one** link to a CAV is mandated. However, I would argue that without an explicit label or an explicit description in the overlay interface what settings users would need to change to produce the CAV, the mechanism cannot be considered accessibility supported (failing [3.3.2 Labels or Instructions](https://www.w3.org/TR/WCAG21/#labels-or-instructions)).

If this argument is the position of the Working Group would adopt, it would follow that **a generic overlay without some explicit signposting how to reach the CAV can never be considered a CAV** even in the (very unlikely) case that the overlay would be able to remedy **all** WCAG failures of a site with some combination of settings.
